component-library/components/sections/Left-right-block--Featured-map/Left-right-block--Featured-map.scss

```diff
@@ -0,0 +1,77 @@
+.c-contact-block {
+    $c: &;
+    display: grid;
+    grid-template-columns: var(--twelve-column-grid);
+    row-gap:48px;
+    padding:96px 0 0!important;
```

Comment:
Any chance this can be added to the content so that we don't have to add !important?

Reply:
Targeted the &.component style as this was what the padding was being set on... removes !important and works as expected